src/components/Containers/TopSection/TopSection.jsx

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import styled from "styled-components";
+
+import SideGraphic from "../../../assets/top-section-graphic.svg";
+import { media } from "../../../utils";
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+
+export const TopSection = ({ ...props }) => (
+  <MainContainer {...props}>
+    <MobileGraphic>
+      <Graphic src={SideGraphic} alt="Graphic" />
+    </MobileGraphic>
+    <FirstColumn>
+      <JosefinTitle size="xxlarge" bold="true">
+        Prospective Medical Professionals
+      </JosefinTitle>
+      <SText size="default">
+        A registered charity dedicated to connecting aspiring students to health
+        sciences opportunities
+      </SText>
+      <SButton>
+        <ButtonText size="default" bold="true">
+          DONATE
+        </ButtonText>
+      </SButton>
+    </FirstColumn>
+    <SecondColumn>
+      <Graphic src={SideGraphic} alt="Graphic" />
+    </SecondColumn>
+  </MainContainer>
+);
+
+const MainContainer = styled.div`
+  display: flex;
+  margin: 5em;
+  ${media(
+    "mobile",
+    `
+        margin: 1em;
+        flex-direction: column;
+        `
+  )};
+`;
+
+const SText = styled(Text)`
+  ${({ theme }) => `
+    font-family: ${theme.font.lato};
+    color: ${theme.colors.navy};
+    margin-top: 0;
+  `};
+  ${media(
+    "mobile",
+    `
+        line-height: 1.3;
+        `
+  )};
+`;
+
+const JosefinTitle = styled(SText)`
+  ${({ theme }) => `
+    font-family: ${theme.font.josefin};
+    margin: 1em 0 0 0;
+  `};
+  ${media(
+    "mobile",
+    `
+        font-size: 2.5rem;
+        `
+  )};
+`;
+
+const ButtonText = styled(SText)`
+  margin: 0 0 0 0;
+`;
+
+const Graphic = styled.img`
```

Comment:
Should be a styled.svg`` since it is a vector image (you can search the difference between svg and jpg)

Reply:
Won't really be able to do this. To use the `svg` tag would require inlining all of the svg code in the file - pretty bad idea (browser can't cache the graphic, other issues). `<img>` tags seem to be recommended for displaying vector graphics from my research (incorporates ability to use fallback images for older browsers easily, though we don't really have those in the project yet)

Reply:
Ah I didn't know too much about the difference of ```img``` and ```svg``` html tags in terms of caching, but I took a few minutes to read about it and it makes sense. We'll keep your changes then. Thanks for taking the time to review it!